man/networkctl.xml

```diff
@@ -405,6 +430,25 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--drop-in=</option></term>
```

Comment:
Add `<replaceable>NAME</replaceable>` here, and below say "edit the drop-in file `<replaceable>NAME</replaceable>` instead of the main configuration file".